json/web-check.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Web-Check",
+  "slug": "web-check",
+  "categories": [
+    4
+  ],
+  "date_created": "2025-02-26",
+  "type": "ct",
+  "updateable": false,
+  "privileged": false,
+  "interface_port": 3000,
+  "documentation": "https://github.com/Lissy93/web-check/blob/master/.github/README.md",
+  "website": "https://github.com/lissy93/web-check",
+  "logo": "https://github.com/Lissy93/web-check/blob/master/public/favicon-32x32.png?raw=true",
```

Is this logo not better?
```suggestion
  "logo": "https://raw.githubusercontent.com/Lissy93/web-check/refs/heads/master/.github/web-check-logo.png",
```